contracts/utils/Address.sol

```diff
@@ -24,10 +45,14 @@ library Address {
      * https://solidity.readthedocs.io/en/v0.8.0/security-considerations.html#use-the-checks-effects-interactions-pattern[checks-effects-interactions pattern].
      */
     function sendValue(address payable recipient, uint256 amount) internal {
```

Comment:
It looks that this is could be rewritten as

```solidity
    function sendValue(address payable recipient, uint256 amount) internal {
        functionCallWithValue(recipient, new bytes(0), amount);
    }
```

Reply:
Neeat!

Reply:
Reverted back with the same reasoning as in https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4261#discussion_r1212498777

Reply:
Send value should allow sending to non-contract addressed, `functionCallWithValue` uses the `verifyCallResultFromTarget` which checks against EOAs.